osquery/tables/events/linux/apparmor_events.cpp

```diff
@@ -0,0 +1,109 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+#include <osquery/registry_factory.h>
+
+#include "osquery/events/linux/auditeventpublisher.h"
+#include "osquery/tables/events/linux/apparmor_events.h"
+
+namespace osquery {
+FLAG(bool,
+     audit_allow_apparmor_events,
+     false,
+     "Allow the audit publisher to process audit events");
```

Comment:
This isn't clear

Reply:
Yeah, just a typo. Secont `audit` should be `AppArmor`. 

Thanks!